### ISSUE TITLE
Whisper fails to connect

### ISSUE THREAD

**Issue by rifting** [MercuryWorkshop/wisp-server-node#3]
Expected behavior:
Whisper will connect to the websocket and the host system will send all traffic through the TUN interface.

Actual behavior: Whisper errors out and crashes the websocket server, wss://nebulaproxy.io/wisp/ for example:

Whisper logs:
```
whisper.exe --tun tun100 --url wss://nebulaproxy.io/wisp/
Connecting to WebSocket: wss://nebulaproxy.io/wisp/
Creating TUN device with name: "tun100"
Error in Wisp multiplexor future: Websocket implementation error: Unexpected EOF
```

Nebula logs:
```
UDP socket bound to :::53482
UDP socket bound to :::53483
UDP send error: Error: getaddrinfo ENOTFOUND EARCH * HTTP/1.1
Host: [FF02::C]:1900
ST: urn:dial-multiscreen-org:service:dial:1
Man: "ssdp:discover"
MX: 3


    at __node_internal_captureLargerStackTrace (node:internal/errors:563:5)
    at __node_internal_ (node:internal/errors:782:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)
    at GetAddrInfoReqWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'EARCH * HTTP/1.1\r\n' +
    'Host: [FF02::C]:1900\r\n' +
    'ST: urn:dial-multiscreen-org:service:dial:1\r\n' +
    'Man: "ssdp:discover"\r\n' +
    'MX: 3\r\n' +
    '\r\n'
}
UDP send error: Error: getaddrinfo ENOTFOUND EARCH * HTTP/1.1
Host: [FF02::C]:1900
ST: urn:dial-multiscreen-org:service:dial:1
Man: "ssdp:discover"
MX: 3

M-SEARCH * HTTP/1.1
Host: [FF02::C]:1900
ST: uuid:5f24d1c3-e6e7-c3cd-4ea6-5c76238fa9af
Man: "ssdp:discover"
MX: 3


    at __node_internal_captureLargerStackTrace (node:internal/errors:563:5)
    at __node_internal_ (node:internal/errors:782:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)
    at GetAddrInfoReqWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'EARCH * HTTP/1.1\r\n' +
    'Host: [FF02::C]:1900\r\n' +
    'ST: urn:dial-multiscreen-org:service:dial:1\r\n' +
    'Man: "ssdp:discover"\r\n' +
    'MX: 3\r\n' +
    '\r\n' +
    'M-SEARCH * HTTP/1.1\r\n' +
    'Host: [FF02::C]:1900\r\n' +
    'ST: uuid:5f24d1c3-e6e7-c3cd-4ea6-5c76238fa9af\r\n' +
    'Man: "ssdp:discover"\r\n' +
    'MX: 3\r\n' +
    '\r\n'
}
About to throw: Not running
c:\Users\\Downloads\NebulaNew\Nebula\node_modules\.pnpm\@github.com+NebulaServices+rammerhead+releases+download+rammerhead-1.2.41-nebula.8@rammerhead_srd7prfi5xinpefqtjlkw4cz34\node_modules\rammerhead\src\util\addMoreErrorGuards.js:35  
        throw err;
        ^


Error [ERR_SOCKET_DGRAM_NOT_RUNNING]: Not running
    at healthCheck (node:dgram:919:11)
    at Socket.close (node:dgram:744:3)
    at <anonymous> (c:\Users\\Downloads\NebulaNew\Nebula\node_modules\.pnpm\wisp-server-node@1.0.5\node_modules\wisp-server-node\dist\ConnectionHandler.js:129:47)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21) {
  code: 'ERR_SOCKET_DGRAM_NOT_RUNNING'
}


```

**Comment by wspyams** [comment]
thanks for letting us know! i believe an idea on how to fix this. will be pushed once i get home

**Comment by ProgrammerIn-wonderland** [comment]
Your acting like we weren't all just talking about this bug over discord 😭

**Comment by wspyams** [comment]
nuh uh

**Comment by wspyams** [comment]
![watermark](https://github.com/MercuryWorkshop/wisp-server-node/assets/92237104/6a9c0528-1743-4cbb-9295-87f93ec471e8)

**Comment by wspyams** [comment]
fixed with [commit c49fbb9](https://github.com/MercuryWorkshop/wisp-server-node/commit/c49fbb9b56b451e2ea88d358f02e0b760e708e70)
